### ISSUE TITLE
Include instead of Exclude?

### ISSUE THREAD

**Issue by RussellBishop** [abynim/sketch-selective-upload#3]
I would love to be able to swap the logic around so that I can use Include instead of Exclude.

This means I can just grab one Page, add the Prefix, and Upload.

**Comment by RussellBishop** [comment]
![Screen Shot 2020-04-28 at 14 09 30](https://user-images.githubusercontent.com/866276/80490871-e547bd00-8959-11ea-8644-b6840854d3af.png)

What it looks like right now… :(

**Comment by abynim** [comment]
Interesting. So in an ideal scenario you would set ⚡ as the `include` prefix and only that page gets uploaded when you sync to cloud? 

Nice use of emoji as prefixes by the way 👍

**Comment by RussellBishop** [comment]
Exactly! Then I can just do tiny uploads to sketch.cloud really quickly ♥️

**Comment by abynim** [comment]
Cool, will think about it. Would you personally use this on a per document level or as a global preference?

Technically it's quite easy to implement, but it's more of a UX challenge to ensure people know what's happening. We don't want folks to send angry emails to Sketch support saying their documents aren't being uploaded (because they forgot to add an `include` prefix)

**Comment by RussellBishop** [comment]
For me this would be a global preference.

Totally see your point about `include` sending blank docs – could you use an alert here?

'No artboards or pages were prefixed with ⚡️ to upload to Sketch Cloud.

**Comment by DjangoBlok** [comment]
I would like the same logic. Only upload "+" prefixed pages.
